modules/cnf-node-tuning-operator-creating-pod-with-guaranteed-qos-class.adoc

```diff
@@ -6,13 +6,51 @@
 [id="cnf-node-tuning-operator-creating-pod-with-guaranteed-qos-class_{context}"]
 = Creating a pod with a guaranteed QoS class
 
-Keep the following in mind when you create a pod that is given a QoS class of `Guaranteed`:
+You can create a pod with a quality of service (QoS) class of `Guaranteed` for high-performance workloads. Configuring a pod with a QoS class of `Guaranteed` ensures that the pod has priority access to the specified CPU and memory resources. 
 
-* Every container in the pod must have a memory limit and a memory request, and they must be the same.
-* Every container in the pod must have a CPU limit and a CPU request, and they must be the same.
+To create a pod with QoS class of `Guaranteed`, you must apply the following specifications:
 
-The following example shows the configuration file for a pod that has one container. The container has a memory limit and a memory request, both equal to 200 MiB. The container has a CPU limit and a CPU request, both equal to 1 CPU.
+* Set identical values for the memory limit and memory request fields for each container in the pod.
+* Set identical values for CPU limit and CPU request fields for each container in the pod.
 
+[NOTE]
+====
+A pod with a QoS class of `Guaranteed` cannot use resources beyond the specified limits.
+====
+
+The pod will not be evicted from a node unless one of the following criteria is met:
+
+* During resource contention on a node, there are no lower-priority pods that can be evicted.
+* The pod exceeds the CPU or memory limits.
```

Comment:
It cannot exceed cpu limits even if it tries (cgroup throttling will not let it).

Reply:
OK, will remove this bullet then